src/main.cpp

```diff
@@ -57,44 +57,25 @@ int findFunction(Uhh& uhh, const std::string name, const std::vector<std::string
     return 0;
 }
 
+#define buildCommand(name, usage, function) {name, usage, function}
```

Comment:
This is the only thing I don't like.  We can fix it in a bit.  

macros, afaik, are typically all upper case to let the user know that its a macro :)

Either way.  I'll merge and change some day.  Some day when the wind blows